cwl/msi.cwl

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: cwl:v1.0
+
+#msisensor:
+#    msisensor msi \
+#    -d msi_sites
+#    -n normal_bam
+#    -t tumor_bam
+#    -tumor_sample_name tumor_sample_name
+#    -normal_sample_name normal_sample_name
+#    -o ${ return inputs.tumor_sample_name +"."+inputs.normal_sample_name+".msi.txt" }
+#
+
+class: CommandLineTool
+baseCommand:
+- "msisensor"
+- "msi"
+
+
+requirements:
+  DockerRequirement:
+    dockerPull: mskcc/msisensor:0.2
+  InlineJavascriptRequirement: {}
+  ResourceRequirement:
+    ramMin: 16
+    coresMin: 2
+
+doc: |
+  Run msisensor on tumor-normal bams to differentiate MSI (microsatellite instable) samples from MSS (microsatellite stable) ones
+
+inputs:
+  d:
+    type:
+    - string
+    - File
+    doc: homopolymer and microsatellites file
+    inputBinding:
+      prefix: -d
+
+  n:
+    type:
+    - File
+    doc: normal bam file
+    secondaryFiles: ["^.bai"]
+    inputBinding:
+      prefix: -n
+
+  t:
+    type:
+    - File
+    doc: tumor bam file
+    secondaryFiles: ["^.bai"]
+    inputBinding:
+      prefix: -t
+
+  o:
+    type: string
+    doc: output distribution file
+    inputBinding:
+      prefix: -o
+
+#below are optional inputs
```

Comment:
I think we should skip inclusion of optional cli args inside the CWL unless we are actually using them in the workflows somewhere. It makes the CWL bloated and confusing. If we are using something like `-f 0.05` all the time then we can just bundle it into the `baseCommand` like `["msisensor", "msi", "-f", "0.05"]`.